lib/va_profile/models/v3/address.rb

```diff
@@ -52,10 +52,7 @@ def in_json_v2
             effectiveEndDate: @effective_end_date
           }
 
-          if @validation_key.present?
-            address_attributes[:validationKey] = @validation_key
-            address_attributes[:overrideIndicator] = true
-          end
+          address_attributes[:overrideValidationKey] = @override_validation_key if @override_validation_key.present?
 
```

Comment:
validationKey  and overrideIndicator was deprecated with ContactInfoV2

Reply:
VAProfile does not accept a null override_validation_key, so we only include it in the request when it has a value.